plugin/v2/settable.go

```diff
@@ -0,0 +1,73 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
```

Comment:
@AkihiroSuda not in this PR, but soon after I'll add `Mounts` support, and in `Mounts` you don't set "value", you set "source" (an maybe "destination" in the future)